### ISSUE TITLE
Course Registration

### ISSUE THREAD

**Issue by wzcy** [mickesv/LSRE-ReleasePlanningProject#179]
As a new student, I shall be able to register courses in the system.

**Comment by wzcy** [comment]
related #182

**Comment by mickesv** [comment]
please elaborate on "register courses". Do you mean "register **on** courses" ?

**Comment by wzcy** [comment]
As a new student, I shall be able to register on courses in the system. It means that I can choose the courses to take for the coming semester from the course options list #182 .

**Comment by mickesv** [comment]
How is this different from #182? Or is it that #182 contains two requirements?

**Comment by wzcy** [comment]
Yes, I mad a mistake. #182 only focus on the new students can have a view of course list which contains the all course introductions. #179 focus on the course registration.
